prov/usnic/src/usdf_domain.c

```diff
@@ -518,9 +517,9 @@ int usdf_check_mr_mode(uint32_t version, const struct fi_info *hints,
 {
 	int ret;
 
-	ret = ofi_check_mr_mode(&usdf_ops, version, hints->caps, prov_mode,
-				hints->domain_attr->mr_mode);
+	ret = ofi_check_mr_mode(&usdf_ops, version, prov_mode, hints);
 
+	/* TODO: Checks below may not be needed */
```

Comment:
@bturrubiates @jsquyres - need confirmation on usnic changes and if more are needed.